docs/cn/compile.md

```diff
@@ -24,6 +28,35 @@
     make install
     ```
 
+# 编译
+
+[build]: build
+
+## 硬件要求
+
+- **内存**: 最小 4GB, 推荐 8GB+.
+- **硬盘**: 全量编译需要至少 25GB 的空闲磁盘空间
+- **操作系统**: 64 位 Linux 或者 macOS >= 10.14 
+
+## 依赖工具
+
+- gcc 8 或更新版本
+- cmake 3.20 或更新版本
+- jdk 8
+- python3, python setuptools, python wheel
+- apache maven 3.x
```

Comment:
maven has to be >= 3.3.9

see [here](https://stackoverflow.com/questions/53211941/unable-to-load-the-mojo-test-org-apache-maven-pluginsmaven-surefire-plugin)